examples/07-EMIT/ComputeProtectionLevels.py

```diff
@@ -0,0 +1,369 @@
+# Configure the radios
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Enable the HR-DSSS bands for the WiFi Radio and set the power level
+# for all Transmit bands to -20 dBm
+def set_band_power_level(band, power):
+    """Set the power of the fundamental for the given band.
+    Arguments:
+        band: Band being configured.
+        power: Peak amplitude of the fundamental [dBm].
+    """
+    prop_list = { "FundamentalAmplitude": power}
+    for child in band.children:
+        if child.props["Type"] == "TxSpectralProfNode":
+            child._set_prop_value(prop_list)
+            return # only one Tx spectral profile per band
+        
+bands = wifi.bands()
+for band in bands:
+    if "HR-DSSS" in band.node_name:
+        if "Ch 1-13" in band.node_name:
+            band.enabled=True
+            set_band_power_level(band, "-20")
+
+# Reduce the bluetooth transmit power
+bands = bluetooth.bands()
+for band in bands:
+    set_band_power_level(band, "-20")
+
+# Configure the first Rx band in the GPS Rx to have 0 dBm Susc
+def set_protection_band(radio):
+    """Set susceptibility of the Rx Band to 0 dBm
+    for all frequencies.
+    Arguments:
+        radio: Radio to modify.
+    """
+    bands = radio.bands()
+    prop_list = {
+                "InBandSensitivity": "0",
+                "SnrAtSensitivity": "0",
+                "RxMaxAttenuation": "0"
+                }
+    for band in bands:
+        for child in band.children:
+            if child.props["Type"] == "RxSusceptibilityProfNode":
+                child._set_prop_value(prop_list)
+                break # only one Rx Spectral Profile per Band
+
+def get_radio_node(radio_name):
+    """Get the radio node that matches the
+    given radio name.
+    Arguments:
+        radio_name: String name of the radio.
+    Returns: Instance of the radio.
+    """
+    if gps.name == radio_name:
+        radio = gps
+    elif bluetooth.name == radio_name:
+        radio = bluetooth
+    else:
+        radio = wifi
+    return radio
+set_protection_band(bluetooth)            
+set_protection_band(gps)
+set_protection_band(wifi)
+
+bands = gps.bands()
+for band in bands:
+    for child in band.children:
+        if "L2 P(Y)" in band.node_name:
+            band.enabled=True
+        else:
+            band.enabled=False
+
+    
+###############################################################################
+# Load the results set
+# ~~~~~~~~~~~~~~~~~~~~
+# Create a results revision and load it for analysis
+
+rev = emitapp.analyze()
+modeRx = emitapp.tx_rx_mode().rx
+modeTx = emitapp.tx_rx_mode().tx
+modeEmi = emitapp.result_type().emi
+
+###############################################################################
+# Generate a legend
+# ~~~~~~~~~~~~~~~~~
+# Define the thresholds and colors used to display the results of 
+# the protection level analysis.
+
+def create_legend_table():    
+    """Creates a table showing the defined protection levels."""
```

Comment:
```suggestion
    """Create a table showing the defined protection levels."""
```
Descriptions for methods/functions start with a verb with an "s" ending. Descriptons for classes to have an "s" ending.